lite/provider_test.go

```diff
@@ -1,37 +1,58 @@
-package certifiers_test
+// nolint: vetshadow
+package lite_test
 
 import (
 	"testing"
 
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 
-	"github.com/tendermint/tendermint/certifiers"
-	"github.com/tendermint/tendermint/certifiers/errors"
+	"github.com/tendermint/tendermint/lite"
+	liteErr "github.com/tendermint/tendermint/lite/errors"
 )
 
+// missingProvider doens't store anything, always a miss
+// Designed as a mock for testing
+type missingProvider struct{}
+
+// NewMissingProvider returns a provider which does not store anything and always misses.
+func NewMissingProvider() lite.Provider {
+	return missingProvider{}
+}
+
+func (missingProvider) StoreCommit(_ lite.FullCommit) error { return nil }
```

Comment:
I think you can drop `_ ` part

Reply:
Damn, you are right. I didn't know that.